integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkCommonIT.java

```diff
@@ -679,6 +679,12 @@ protected String getCreateSimpleTableString(String tableName) {
         tableName);
   }
 
+  protected String getCreateExternalTableString(String tableName) {
```

Comment:
suggest to do some abstraction with `getCreateSimpleTableSql`.

Reply:
How about declare an abstract method in `SparkEnvIT.java`
` protected abstract String getCreateSimpleTableString(String... args);`

and the implementation in `SparkHiveCatalogIT.java `

```
@Override
  protected String getCreateSimpleTableString(String... args) {
    if(args.length == 1) {
      return String.format(
              "CREATE TABLE %s (id INT COMMENT 'id comment', name STRING COMMENT '', age INT)",
              args[0]);
    } else {
      return String.format(
              "CREATE %s TABLE %s (id INT COMMENT 'id comment', name STRING COMMENT '', age INT)",
              args[0], args[1]);
    }
  }
```

Reply:
defining `tableName` and `isExternal` seems more clear.
```java
  protected String getCreateSimpleTableString(String tableName) {
    return getCreateSimpleTableString(tableName, false);
  }

  protected String getCreateSimpleTableString(String tableName, boolean isExternal) {
   // generate create table SQL
 }

```